pacu/modules/apigateway__enum/main.py

```diff
@@ -141,43 +151,50 @@ def parse_method(base_url, method, path, stages):
     return api_method
 
 
-def main(args, pacu):
+def main(args, pacu_main: "Main"):
     """Main module function, called from Pacu"""
-    print = pacu.print
-    session = pacu.get_active_session()
+    session = pacu_main.get_active_session()
+    print = pacu_main.print
     args = parser.parse_args(args)
 
     outfile_path = downloads_dir()/'apigateway'
```

Comment:
We should remove all file output stuff if this data is now saved in the DB

Reply:
Stand by, I think I missed something on the database. Digging into this now and will also fix this

Reply:
@DaveYesland -- This should all be fixed now